source/NVDAObjects/window/winword.py

```diff
@@ -1307,6 +1307,22 @@ def script_toggleAlignment(self,gesture):
 		if msg:
 			ui.message(msg)
 
+	@script(gestures=["kb:control+m", "kb:control+shift+m", "kb:control+t", "kb:control+shift+t"])
+	def script_changeParagraphLeftIndent(self, gesture):
+		if not self.WinwordSelectionObject:
+			# We cannot fetch the Word object model, so we therefore cannot report the format change.
+			# The object model may be unavailable because this is a pure UIA implementation such as Windows 10 Mail,
+			# or it's within Windows Defender Application Guard.
+			# For now, just let the gesture through and don't report anything.
+			return gesture.send()
+		margin = self.WinwordDocumentObject.PageSetup.LeftMargin
+		val = self._WaitForValueChangeForAction(
+			lambda: gesture.send(),
+			lambda: self.WinwordSelectionObject.paragraphFormat.LeftIndent
+		)
+		msg = self.getLocalizedMeasurementTextForPointSize(margin + val)

```

Comment:
```suggestion
		margin = self.WinwordDocumentObject.PageSetup.LeftMargin
		oldVal = self.WinwordSelectionObject.paragraphFormat.LeftIndent
		val = self._WaitForValueChangeForAction(
			lambda: gesture.send(),
			lambda: self.WinwordSelectionObject.paragraphFormat.LeftIndent
		)
		if oldVal == val:
			# indentation hasn't changed, it's likely the gesture used is not mapped to an indentation
			# change in Word, which is the case for some languages (see #6269)
			return
		msg = self.getLocalizedMeasurementTextForPointSize(margin + val)
```

Reply:
Localizing the keystrokes is the work of translators (edition of the gestures.ini file).
I am a bit reluctant to add this piece of code that is useful only in case a translator has not done the required translation.

Note also that this strategy is not the one used for other formatting commands. For example, not all languages use control+B to toggle bold formatting, e.g. French and Italien use ctrl+G (for "Gras" / "Grassetto").

Reply:
Ah, my mistake. LGTM then